torch/autograd/gradcheck.py

```diff
@@ -1273,7 +1278,11 @@ def _gradcheck_helper(func, inputs, eps, atol, rtol, check_sparse_nnz, nondet_to
 
     gradcheck_fn = _fast_gradcheck if fast_mode else _slow_gradcheck
     _gradcheck_real_imag(gradcheck_fn, func, func_out, tupled_inputs, outputs, eps,
-                         rtol, atol, check_grad_dtypes, check_forward_ad=check_forward_ad, nondet_tol=nondet_tol)
+                         rtol, atol, check_grad_dtypes, check_forward_ad=check_forward_ad,
+                         check_backward_ad=check_backward_ad, nondet_tol=nondet_tol)
+    # Short circuit because remaining tests rely on backward AD to be implemented
```

Comment:
We should have an assert that `not (check_undefined_grad and not check_backward_ad)`

Reply:
Probably for check_batched_grad as well?

Reply:
Sure